network.go

```diff
@@ -336,6 +336,118 @@ func (s *sandbox) listInterfaces(netHandle *netlink.Handle) (*pb.Interfaces, err
 	return &interfaces, nil
 }
 
+///////////////
+// Neighbors //
+///////////////
+
+func (s *sandbox) deleteNeighs(netHandle *netlink.Handle, link int) error {
+	if netHandle == nil {
+		return errNoHandle
+	}
+
+	initNeighList, err := netHandle.NeighList(link, netlink.FAMILY_ALL)
+	if err != nil {
+		return err
+	}
+
+	for _, initNeigh := range initNeighList {
+		// don't delete Neighbours associated with lo:
+		link, _ := netHandle.LinkByIndex(initNeigh.LinkIndex)
+		if link.Attrs().Name == "lo" || link.Attrs().Name == "::1" {
+			continue
+		}
+
+		err = netHandle.NeighDel(&initNeigh)
```

Comment:
maybe add initNeigh := initNeigh since initNeigh is loop variable and does not get reinstantiated in loop.